oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/AnalyzerModuleConfig.java

```diff
@@ -134,7 +134,7 @@ public boolean shouldIgnorePeerIPDue2Virtual(int componentId) {
     }
 
     /**
-     * Get all files could be meter analyzed, files split by ","
+     * Get all files could be metered analyzed, files split by ","
```

Comment:
```suggestion
     * Get all meter analysis script files, files split by ","
```

Reply:
I have found this one at present, I'll find typos or grammar again if I have time

Reply:
I would recommend you to fix those when you have several. Otherwise, if you continue to do this randomly, the commit logs will be mass.

Reply:
ok